Userland/Libraries/LibURL/Parser.cpp

```diff
@@ -1688,10 +1688,12 @@ URL Parser::basic_parse(StringView raw_input, Optional<URL> const& base_url, URL
             break;
         // -> query state, https://url.spec.whatwg.org/#query-state
         case State::Query:
-            // FIXME: 1. If encoding is not UTF-8 and one of the following is true:
+            // 1. If encoding is not UTF-8 and one of the following is true:
             //           * url is not special
```

Comment:
small nitpick, could we also un-indent the rest of the comment by 6, otherwise code LGTM

Reply:
done